### PR TITLE
Adds credit link check

### PR DESCRIPTION
**Pull request by danielwrobert**
Added check to see if WP credit link is present in the footer.

### CODE REVIEW

File: components/project/content-full.php [hunk 1]
```diff
@@ -24,6 +24,9 @@
 			<dt><?php esc_html_e( 'Type of project', 'strikebase' ); ?></dt>
 			<dd><?php strikebase_show_project_type( get_the_ID() ); ?></dd>
 
+			<dt><?php esc_html_e( 'WordPress Credit Link', 'strikebase' ); ?></dt>
+			<dd><?php echo ( strikebase_has_credit_link( get_the_ID() ) ? 'Yes' : 'No' ); ?></dd>
```
Comment (ChrisHardie) [inline components/project/content-full.php]: Even though the current output is safe, still best to add an escaping function here, maybe `esc_html()`.

File: inc/template-tags.php [hunk 2]
```diff
@@ -60,6 +60,25 @@ function strikebase_get_project_meta( $post_ID, $key ) {
 	endif;
 }
 
+/*
+ * Outbot for WP Credit Link checkbox
```
Comment (ChrisHardie) [inline inc/template-tags.php]: Outbot -> Output ?

File: components/project/content-full.php (hunk 1 above)
Comment (danielwrobert) [reply components/project/content-full.php]: Agreed, nice catch.

**Comment by danielwrobert** [comment]
@ChrisHardie made updates based on your findings. Look good to merge now?

**Comment by ChrisHardie** [comment]
@danielwrobert go for it!

**Pull request merged**